stl/inc/xutility

```diff
@@ -5503,43 +5405,6 @@ struct uses_allocator
 
 template <class _Ty, class _Alloc>
 _INLINE_VAR constexpr bool uses_allocator_v = uses_allocator<_Ty, _Alloc>::value;
-
-// STRUCT TEMPLATE iterator
-template <class _Category, class _Ty, class _Diff = ptrdiff_t, class _Pointer = _Ty*, class _Reference = _Ty&>
-struct _CXX17_DEPRECATE_ITERATOR_BASE_CLASS iterator { // base type for iterator classes

```

Comment:
I predict that moving `iterator` will wreak the most havoc in RWC. Let's see if I'm right! :-)

(No change requested.)

Reply:
I think you're right too -- I can't think of another reason we would have left it in `<xutility>` back in 2019 RTM. But maybe people have gotten better at removing references to that in that time :).

Reply:
I'm just going to be paranoid and move it back. It's only one struct and no functions unlike the other stuff I demoted.